ununifi/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "ununifi",
+  "assets": [
+    {
+      "description": "The governance token of UnUniFi protocol.",
+      "denom_units": [
+        {
+          "denom": "uguu",
+          "exponent": 0
+        },
+        {
+          "denom": "guu",
+          "exponent": 6
+        }
+      ],
+      "base": "uguu",
+      "name": "UnUniFi",
+      "display": "guu",
+      "symbol": "GUU",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/ununifi/images/ununifi.svg",
```

Comment:
Can we please remove the svg with degraded quality and black (instead of transparent) background? (Can still be included, but shouldn't be referenced here, because frontends will opt for the svg and it will not look as good)

Reply:
I changed to a transparent background svg. Please check.